tests/printer/typexpr/expected/variant.res.txt

```diff
@@ -115,3 +115,11 @@ type t = [
 
 // don't pick int with suffix as numeric polyvar
 type t = [#"10s" | #"20t"]
+
+// comments
+type foo = [
+  | // before bar
```

Comment:
@IwanKaramazow Not that easily I think because the `|` comes from the `Doc.join` invocation:
```ocaml
let cases = Doc.join ~sep:(Doc.concat [Doc.line; Doc.text "| "]) docs in
```